### DISCUSSION TITLE
Is there a way to check which breakpoint is active in react-spectrum?

### DISCUSSION THREAD

**Discussion by mistyharsh** [adobe/react-spectrum#6973] [Q&A]
The `useProvider()` provides breakpoint object. However, I need to know which breakpoint is/are active. Is there any out-of-the-box utility in `@adobe/react-spectrum` to get this? Or, would I need to write my custom media query using the values from breakpoint object?

**Comment by reidbarber** [comment]
Not at the moment: https://github.com/adobe/react-spectrum/issues/2989
